applications/GeoMechanicsApplication/tests/fixed_spatial_variation/simple_mesh.mdpa

```diff
@@ -0,0 +1,104 @@
+Begin Table 1 X WATER_PRESSURE
+    0.0000000000 0.0000000000
+    1.0000000000 -5000
+End Table
+
+Begin Table 2 X WATER_PRESSURE
+    0.0000000000 -40000
+    1.0000000000 -30000
+End Table
+
+Begin Properties 1
+End Properties
+
+
+Begin Nodes
+  1  0.0000000000 -2.0000000000 0.0000000000
+  2  1.0000000000 -2.0000000000 0.0000000000
+  3  1.0000000000 0.0000000000 0.0000000000
+  4  0.0000000000 0.0000000000 0.0000000000
+  5  0.5000000000 -2.000000000 0.0000000000
+  6  1.0000000000 -1.000000000 0.0000000000
+  7  0.5000000000 0.0000000000 0.0000000000
+  8  0.0000000000 -1.000000000 0.0000000000
+End Nodes
```

Comment:
Really nagging, but for human readers faster.
Line up the . on the consecutive lines such that the - does not cause a disturbed view.

Reply:
Done